src/Expander.hs

```diff
@@ -119,6 +124,7 @@ expandModule thisMod src = do
                            }
     bs <- view expanderCurrentBindingTable <$> getState
     modifyState $ set expanderCurrentBindingTable startBindings
+    modifyState $ set expanderDefTypes startDefTypes
```

Comment:
Is the purpose here to remove `defTypes` that were added during the other expansion tasks?

Reply:
The idea is that each module has its own Γ, and recursively loading a dependency shouldn't put the things I've just defined in my own module into scope in the dependency.